pelicanconf.py

```diff
@@ -1,3 +1,27 @@
 # -*- coding: utf-8 -*-
 
+AUTHOR = 'Komitywa'
+SITENAME = 'Komitywa.org'
+SITEURL = ''
+
+THEME = 'themes/notmyidea'
```

Comment:
First of all I think, that we should create new theme at all - not using some theme from pelican distribution and start from draft - I see that `themes` contain two directories - so I believe You want to point here to `themes/komitywa`.